job-dsls/jobs/communityRelease_pipeline.groovy

```diff
@@ -316,7 +316,11 @@ pipeline {
         // the tags of the release will be created and pushed to github
         stage('Push community tag') {
             steps {
-                sh './droolsjbpm-build-bootstrap/script/release/08a_communityPushTags.sh'
+                script {
```

Comment:
@mbiarnes what's the reason and the effect of adding this?

Reply:
the tags thta are created on the node (i.e. 7.50.0.Final) will be pushed to github